product_docs/docs/lasso/4/release-notes.mdx

```diff
@@ -2,182 +2,161 @@
 title: Release notes
 ---
 
-# Lasso - Version 4.11.0 (2023-12-14)
+## Lasso - Version 4.11.0 (2023-12-14)
 
-## New features
-
-- Implement EPAS and 2ndQPG flavour detection (DC-875)
-- Gathering EPAS Catalog - Views related to partitioned tables in Oracle syntax (DC-266)
-- Support for PostgreSQL 16 (DC-887)
-- Add support for RHEL9 (DC-802, DC-803)
-
-## Bug fix
-
-- Fix situation when `ls pg_data` command takes much time to complete (when number of files is extremely large). Adding
-a default `timeout` of `120s` for this cases. (DC-845)
+Lasso Version 4.11.0 includes the following enhancements and bug fixes:
 
+| Type           | Description | Addresses |
+|----------------|-------------|-----------|
+| Feature        | Implemented EDB Postgres Advanced Server and 2ndQPG flavor detection. | DC-875 |
+| Feature        | Now gathering EDB Postgres Advanced Server Catalog - Views related to partitioned tables in Oracle syntax. | (DC-266) |
+| Feature        | Added support for PostgreSQL 16. | DC-887 |
+| Feature        | Added support for RHEL9. | DC-802, DC-803 |
+| Bug fix        | Fixed situation when `ls pg_data` command takes much time to complete with extremely large number of files. Added a default `timeout` of `120s` for this case. | DC-845 |
 
 ## Lasso - Version 4.10.0 (2023-11-07)
```

Comment:
Per https://github.com/EnterpriseDB/docs/blob/develop/docs/agreements/release-notes-guidelines.md we should be putting the date after the heading, e.g.

```
Released: 07 Nov 2023
```

Reply:
I'm going to take the date out of all of the headers and put them in the text.